elements/pl-symbolic-input/pl-symbolic-input.py

```diff
@@ -45,7 +46,13 @@ def render(element_html, data):
         operators = ', '.join(['cos', 'sin', 'tan', 'exp', 'log', 'sqrt', '( )', '+', '-', '*', '/', '^', '**'])
```

Comment:
Is `e` a default constant as well? I guess I think it should be if it isn't.

Reply:
done